tsconfig.json

```diff
@@ -15,4 +15,7 @@
     "forceConsistentCasingInFileNames": false,
     "noFallthroughCasesInSwitch": false,
   },
+    "exclude": [
```

Comment:
@vijayshiyani formatting is wrong. Too many spaces

Reply:
fixed